vast/vast.cpp

```diff
@@ -65,9 +65,15 @@ stable_set<path> get_plugin_dirs(const caf::actor_system_config& cfg) {
     VAST_ERROR_ANON(__func__, "failed to get program path");
   if (const char* home = std::getenv("HOME"))
     result.insert(path{home} / ".local" / "lib" / "vast" / "plugins");
-  if (auto user_dirs
-      = caf::get_if<std::vector<std::string>>(&cfg, "vast.plugin-paths"))
-    result.insert(user_dirs->begin(), user_dirs->end());
+  if (auto dirs = caf::get_if<std::vector<std::string>>( //
+        &cfg, "vast.plugin-dirs"))
+    result.insert(dirs->begin(), dirs->end());
+  if (auto dirs = caf::get_if<std::vector<std::string>>( //
+        &cfg, "vast.plugin-paths")) {
+    VAST_WARNING_ANON(__func__, "encountered deprecated vast.plugin-paths "
+                                "option; use vast.plugin-dirs instead.");
+    result.insert(dirs->begin(), dirs->end());
+  }
```

Comment:
Since we haven't release the plugin API yet, I think we can avoid jumping through the hoop here.

Reply:
I didn't even consider that. It makes sense to completely remove `plugin-paths`.